platform/ui/src/components/modal/Modal.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Modal from 'react-bootstrap-modal';
+
+const CustomModal = ({
+  className,
+  closeButton,
+  backdrop,
+  keyboard,
+  show,
+  large,
+  title,
+  onHide,
+  footer: Footer,
+  header: Header,
+  children: Component,
```

Comment:
Do we think we'll need/use the `large` prop? Are all of these worth passing through? Can we add `propTypes`?

Reply:
Done! thx.